randovania/games/prime1/patcher/randomprime_patcher.py

```diff
@@ -337,7 +337,19 @@ def create_patch_data(self, description: LayoutDescription, players_config: Play
             ctwk_config["morphBallSize"] = 0.3
             ctwk_config["easyLavaEscape"] = True
 
-        # ctwk_config["hudColor"] = [0.1, 0.7, 0.2]
+        if cosmetic_patches.use_hud_color:
+            ctwk_config["hudColor"] = [
+                cosmetic_patches.hud_color[0] / 255,
+                cosmetic_patches.hud_color[1] / 255,
+                cosmetic_patches.hud_color[2] / 255
+            ]
+
+        SUIT_ATTRIBUTES = ["powerDeg", "variaDeg", "gravityDeg", "phazonDeg"]
+        suit_colors = {}
+        for i in range(0,4):
```

Comment:
```python
for attribute, hue_rotation in zip(SUIT_ATTRIBUTES, cosmetic_patches.suit_color_rotations):
```

We could have a `__post_init__` in the CosmeticPatches to test that there's precisely 4 color rotations.
Also wow this would've been a perfect use of `zip(strict=True)`, but that's python 3.10 ☹️